### ISSUE TITLE
Samostatny flag pre nestupnovatelne pridavne mena

### ISSUE THREAD

**Issue by jose1711** [sk-spell/hunspell-sk#3]
Myslim, ze `/Y` suffix by bolo vhodne upravit, kedze nie vsetky pridavne mena sa daju stupnovat. Napr. pridat novy flag, ktory sa aplikuje len na pridavne mena, ktore stupnovatelne su.
Priklad:
```
západoberlínsky/Y (západoberlínska, západoberlínski, západoberlínskych..)
buchnutý/YNs (buchnutá, buchnutejší, nebuchnuté)
```

**Comment by zdenop** [comment]
O akom počte slov sa bavíme?  Máš nejakú analýzu?
Par poznámok.
1. To že slovom sa nepoužíva, lebo je divné, neznamená, že je chybné/preklep.
2. Ak aj slovník "generuje" chybné slovo, ktoré je nepravdepodobné, že ho niekto použije, je lepšie to nechať tak snahu o dokonalosť. Osobitne ak je zámer vygenerovať z hunspell slovníka aspell slovník (pokiaľ sa dobre pamätám, tak ten má limit na počty FLAGOV)

**Comment by jose1711** [comment]
nie, ziadne cisla v ruke nemam. prirucka slovenskeho jazyka vravi, ze stupnovat sa daju len akostne prid. mena s vynimkou tych, ktore maju absolutny priznak (a ako to uz v nasom jazyku chodi - aj tam su vynimky): teda živý, mŕtvy, nahý, chorý a pod.

hunspell momentalne povoluje aj stupnovanie privlastnovacich prid. mien odvodenych od zvierat, teda tvary ako mačacejší, pávejší, krokodílejší alebo vztahovych pridavnych mien - teda napr. včerajšejší, učiteľskejší.

**Comment by zdenop** [comment]
O.K. Navrhujem teda nasledovný postup:
1. Poďme vyčistiť sk-sk.dic (priradiť FLAGY, kde sa dá presunúť otázne veci do samostatného súboru)
2. affix kompresiu upravovať až potom (je to veľmi náročné, niekedy pridanie/oprava jedného pravidla veci viac pokazí ako vyrieši)

**Comment by andrewshadura** [comment]
Nie som Slovák, ale myslím si, že ponechať možnosť povedať, že západoberlínsky krokodíl je krokodílejší a západoberlínskejší, ale nie mačacejší, ale pravdepodobne aj vôbec nie pávejší, ako vychodoberlínska mačka :smiley:
